src/main/java/adris/altoclef/tasks/CraftInInventoryTask.java

```diff
@@ -77,13 +83,15 @@ protected Task onResourceTick(AltoClef mod) {
         // No need to free inventory, output gets picked up.
 
             setDebugState("Crafting in inventory... for " + toGet);
+            if(mod.getModSettings().openInvDuringCrafting() && (client == null || client.currentScreen == null || !client.currentScreen.equals(invScreen))) client.setScreen(invScreen);
```

Comment:
This if-statement looks wrong. If client is null, it will try to open the inventory of the null client? Maybe it should be this: (if the settings allow it, and the client is not null, and (either the current screen doesn't exist, or it is not the inventory screen) then open the inventory)
`mod.getModSettings().openInvDuringCrafting() && client != null && (client.currentScreen == null || !client.currentScreen.equals(invScreen))`